torch/csrc/jit/backends/coreml/objc/PTMCoreMLBackend.mm

```diff
@@ -90,6 +90,11 @@ GenericList pack_outputs(const std::vector<TensorSpec>& output_specs, id<MLFeatu
       count * sizeof(float));
     outputs.push_back(tensor);
```

Comment:
```suggestion
    outputs.push_back(std::move(tensor));
```

Reply:
Also the std::vector above it should be call .reserve() to reserve the length since the length is known.